robot/Cumulus/tests/browser/donations/create_contact_donation.robot

```diff
@@ -14,27 +14,26 @@ Suite Teardown  Delete Records and Close Browser
 ***Keywords***
 # Setup a contact with parameters specified
 Setup Test Data
-    &{data} =  Setup Data
-    ...           contact1=&{contact1_fields}
-    Set suite variable    ${contact1_Id}   &{data}[contact1_Id]
+    &{data}=  Setupdata   contact   ${contact1_fields}
+    Set suite variable   &{data}
```

Comment:
I recommend moving the call to `set suite variable` inside of `setupdata`. Since that keyword is specifically for setting up data, it makes sense to not just create the data but also save it as a suite variable. That will make the test just a little cleaner.

Reply:
Thanks, Make sense .Did this change